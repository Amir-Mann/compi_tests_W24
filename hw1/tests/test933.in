O;\%{@O-)p;%]E#>gq
40r&-0<\i9	SW|(SOS/VA$0ap6$s^5c0nA}v<sP.U$v7Xm'd`yi\.3V