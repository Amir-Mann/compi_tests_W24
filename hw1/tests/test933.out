1 ID O
Error 
